task-management/src/main/webapp/js/vuedraggable.js

```diff
@@ -1,2 +1,2 @@
 null;} const e=this.realList[n];return {index: n,element: e};},getUnderlyingPotencialDraggableComponent: function(t){const n=t.__vue__;return n&&n.$options&&n.$options._componentTag==='transition-group'?n.$parent:n;},emitChanges: function(t){const n=this;this.$nextTick(function(){n.$emit('change',t);});},alterList: function(t){if (this.list){t(this.list);} else {const n=S(this.value);t(n),this.$emit('input',n);}},spliceList: function(){const t=arguments,n=function(n){return n.splice.apply(n,S(t));};this.alterList(n);},updatePosition: function(t,n){const e=function(e){return e.splice(n,0,e.splice(t,1)[0]);};this.alterList(e);},getRelatedContextFromMoveEvent: function(t){const n=t.to,e=t.related,r=this.getUnderlyingPotencialDraggableComponent(n);if (!r) {return {component: r};} const o=r.realList,u={list: o,component: r};if (n!==e&&o&&r.getUnderlyingVm){const c=r.getUnderlyingVm(e);if (c) {return i()(c,u);}} return u;},getVmIndex: function(t){const n=this.visibleIndexes,e=n.length;return t>e-1?e:n[t];},getComponent: function(){return this.$slots.default[0].componentInstance;},resetTransitionData: function(t){if (this.noTransitionOnDrag&&this.transitionMode){const n=this.getChildrenNodes();n[t].data=null;const e=this.getComponent();e.children=[],e.kept=void 0;}},onDragStart: function(t){this.context=this.getUnderlyingVm(t.item),t.item._underlying_vm_=this.clone(this.context.element),R=t.item;},onDragAdd: function(t){const n=t.item._underlying_vm_;if (void 0!==n){Object(M['d'])(t.item);const e=this.getVmIndex(t.newIndex);this.spliceList(e,0,n),this.computeIndexes();const r={element: n,newIndex: e};this.emitChanges({added: r});}},onDragRemove: function(t){if (Object(M['c'])(this.rootContainer,t.item,t.oldIndex),t.pullMode!=='clone'){const n=this.context.index;this.spliceList(n,1);const e={element: this.context.element,oldIndex: n};this.resetTransitionData(n),this.emitChanges({removed: e});} else {Object(M['d'])(t.clone);}},onDragUpdate: function(t){Object(M['d'])(t.item),Object(M['c'])(t.from,t.item,t.oldIndex);const n=this.context.index,e=this.getVmIndex(t.newIndex);this.updatePosition(n,e);const r={element: this.context.element,oldIndex: n,newIndex: e};this.emitChanges({moved: r});},updateProperty: function(t,n){t.hasOwnProperty(n)&&(t[n]+=this.headerOffset);},computeFutureIndex: function(t,n){if (!t.element) {return 0;} const e=S(n.to.children).filter(function(t){return t.style['display']!=='none';}),r=e.indexOf(n.related),o=t.component.getVmIndex(r),i=e.indexOf(R)!==-1;return i||!n.willInsertAfter?o:o+1;},onDragMove: function(t,n){const e=this.move;if (!e||!this.realList) {return !0;} const r=this.getRelatedContextFromMoveEvent(t),o=this.context,u=this.computeFutureIndex(r,t);i()(o,{futureIndex: u});const c=i()({},t,{relatedContext: r,draggedContext: o});return e(c,n);},onDragEnd: function(){this.computeIndexes(),R=null;}}};typeof window!=='undefined'&&'Vue'in window&&window.Vue.component('draggable',N);const U=N;n['default']=U;}})['default'];});
```

Comment:
You should avoid linting a js library and BTW, it would be better to define it in commons or gatein-portal project to be able to reuse it + a small announcement/information about this new third party library. ( @mkrout )